.tekton/cert-manager-istio-csr-1-15-push.yaml

```diff
@@ -32,7 +32,7 @@ spec:
     value: .
   - name: build-args
     value:
-      - "RELEASE_VERSION=v1.15.0"
+      - "RELEASE_VERSION=v0.12.0"
```

Comment:
Can elaborate this version usage?

Reply:
It is the release version of the `istio-csr` used and the value will be used for adding `version` and `release` labels in the container image built.